lib/vendorCargoRegistries.nix

```diff
@@ -1,4 +1,5 @@
-{ downloadCargoPackage
+{ crateRegistries
```

Comment:
Just realized this is no longer needed. I used this before getting registries from `lockPackages`.

Reply:
Good catch! Fixed in b7f607c2004246d2784f4a58f60a5840066a872b